src/commands/new/glee.ts

```diff
@@ -1,55 +1,173 @@
-import {Flags} from '@oclif/core';
+import { Flags } from '@oclif/core';
 import { promises as fPromises } from 'fs';
 import Command from '../../base';
 import { resolve, join } from 'path';
 import fs from 'fs-extra';
-
+import { Specification, load } from '../../models/SpecificationFile';
+import path from 'path';
+import yaml from 'js-yaml';
+import { prompt } from 'inquirer';
 export default class NewGlee extends Command {
   static description = 'Creates a new Glee project';
 
   protected commandName = 'glee';
 
   static flags = {
     help: Flags.help({ char: 'h' }),
-    name: Flags.string({ char: 'n', description: 'name of the project', default: 'project' }),
-    template: Flags.string({ char: 't', description: 'name of the template', default: 'default' }),
+    name: Flags.string({
+      char: 'n',
+      description: 'Name of the Project',
+      default: 'project',
+    }),
+    template: Flags.string({
+      char: 't',
+      description: 'Name of the Template',
+      default: 'default',
+    }),
+    file: Flags.string({
+      char: 'f',
+      description:
+        'The path to the AsyncAPI file for generating a Glee project.',
+    }),
   };
 
+  async getFilteredServers(serversObject: any) {
+    const servers = Object.keys(serversObject);
+
+    const localServers = await prompt([
+      {
+        name: 'LOCAL_SERVERS',
+        message:
+          'Select all of the servers that you want glee to set up and run a server for (local servers):',
+        type: 'checkbox',
+        choices() {
+          return servers;
+        },
+      },
+    ]);
+
+    return servers.filter(
+      (server) => !localServers.LOCAL_SERVERS.includes(server)
+    );
+  }
+  async createTemporaryFile(
+    asyncapiInput: Specification,
+    filteredRemoteServers: string[],
+    file: any
+  ) {
+    const asyncapiObject = asyncapiInput.toJson();
+    asyncapiObject['x-remoteServers'] = filteredRemoteServers;
+
+    delete asyncapiObject.filePath;
+    delete asyncapiObject.kind;
+
+    const updatedAsyncApiContent = yaml.dump(asyncapiObject, {
+      lineWidth: -1,
+    });
+
+    const currentFileDirectory = path.join(__dirname, file);
+
+    fs.writeFileSync(currentFileDirectory, updatedAsyncApiContent);
+    return currentFileDirectory;
+  }
+  async handleGenerateProjectWithFile(
+    file: any,
+    CURRENT_GLEE_TEMPLATE: any,
+    projectName: string
+  ) {
+    try {
+      const asyncapiInput = (await load(file)) || (await load());
```

Comment:
@ayushnau 
this load logic will not work if the file is not in the current directory. for example, if I run it with: 
```asyncapi new glee -f ./some/random/folder/asyncapi.yml```
generator will not be able to find the file. 🤔

Reply:
now it is working.